staging/src/k8s.io/apiserver/pkg/util/webhook/webhook_test.go

```diff
@@ -197,7 +197,7 @@ MIIDGTCCAgGgAwIBAgIUOS2M
 				},
 			},
 			user:     &defaultUser,
-			errRegex: "unable to load root certificates: failed to parse certificate: asn1: syntax error: data truncated",
+			errRegex: "unable to load root certificates: failed to parse certificate: (asn1: syntax error: data truncated|x509: malformed certificate)",
```

Comment:
I was going to complain about a message, but honestly I couldn't say what it meant before either ;)